### DISCUSSION TITLE
Numbered lines/paragraphs are automatically converted to lists

### DISCUSSION THREAD

**Discussion by hurzl** [typst/typst#4239] [General]
### Description

All three blocks type lists, but the first block should be treated as single paragraph and the second as ordinary paragraphs, the third even starts counting at 4:

```
1. abc
2. def
3. ghi

1. abc

2. def

3. ghi

+ abc
+ def
+ ghi
```

### Typst version

- [X] I am using the latest version of Typst

**Comment by Enivex** [comment]
This is intentional behavior

**Comment by hurzl** [comment]
It is not what the guide says. And it is annoying if you want to have paragraphs starting with a number.
Btw. why does it not work with a. b. c. A.B.C  etc.? It is confusing.

**Comment by Enivex** [comment]
> It is not what the guide says. And it is annoying if you want to have paragraphs starting with a number.

I don't know which part of the guide you're referring to, but it doesn't cover every part of the typst syntax.

If you're starting every paragraph with a number, that sounds an awful lot like a list to me.

**Comment by Enivex** [comment]
> Btw. why does it not work with a. b. c. A.B.C  etc.? It is confusing.

That's what the numbering parameter of `enum` is for.

**Comment by MDLC01** [comment]
As already mentioned, this is intentional behavior (this is just [the numbered list syntax](https://typst.app/docs/reference/model/enum/#syntax)).

To prevent that, you can use a backslash as such:

```typ
\1. First paragraph.

\2. Second paragraph.
```

This creates multiple paragraphs instead of a numbered list.

**Comment by hurzl** [comment]
Ok the Latex users' guide only talks about + and -. I thought it is about simplicity.

Another bad side-effect: If any line in my source happens to start with a number, it will be made into a list item.

**Reply by MDLC01** [reply]
It has to specifically start with a number followed by a period, and you can always escape it like I explained above. The same problem happens with any piece of syntax.

**Comment by hurzl** [comment]
So I have to search my text for numbers with period at line start?
Otherwise I can get nonsense like here:

```
#set enum(numbering: "a.")

... this is to be found on page
341. See there.
```

**Comment by Enivex** [comment]
> So I have to search my text for numbers with period at line start?
> Otherwise I can get nonsense like here:
> 
> ```
> #set enum(numbering: "a.")
> 
> ... this is to be found on page
> 341. See there.
> ```
> 

I'd advise against hard-wrapping actual text.

**Comment by hurzl** [comment]
Text broken into lines is the standard. For example it's what you get from pandoc converting from odt.

**Comment by Enivex** [comment]
> Text broken into lines is the standard. For example it's what you get from pandoc converting from odt.

I'd say that pandoc shouldn't do that, then.

Typst's markup mode depends on line breaks.

**Comment by hurzl** [comment]
Yes pandoc should watch for numbers ...
All your examples are with broken lines.

> [Paragraph breaks](https://typst.app/docs/reference/model/parbreak/) work just as they do in LaTeX, just use a blank line.

https://typst.app/docs/guides/guide-for-latex-users/
